foam_wrapper/foam_wrapper.py

```diff
@@ -0,0 +1,158 @@
+""" foam_wrapper module
+
+Wrapper module for OpenFOAM using pythonFlu -python interface
+NOTE: at a moment pythonFlu supports only up to 2.0.1 -version of OpenFOAM
+
+"""
+
+from Foam import ref, man, src
+from simphony.cuds.mesh import Mesh, Point, Face, Edge, Cell
+import inspect
+from Foam.OpenFOAM import fileName, word, IOobject
+from collections import defaultdict
+from Foam.finiteVolume import volScalarField
+from Foam.src.OpenFOAM import *
+
+class FoamWrapper(object):
+
+    def __init__(self, model):
+        self.model = model
+
+    def has_common_vertex(self, facePoints0, facePoints1):
+        for point in facePoints0:
+                if point in facePoints1:
+                    return True
+        return False
+
+    def get_mesh(self, foamCaseName):
+        """ Reads given OpenFoam -mesh from "foamCaseName" -directory and returns SimPhony mesh
+        """
+        argv=["test", "-case", foamCaseName]
+        args = ref.setRootCase(len(argv), argv)
+        runTime = man.createTime(args)
+        foamMesh = man.createMesh(runTime)
+
+        foamPoints = foamMesh.points()
+        foamFaces = foamMesh.faces()
+        foamCells = foamMesh.cells()
+
+        simphonyMesh = Mesh()
```

Comment:
I am curious about this line.  Here `simphonyMesh` (which is returned by this function) is a stand-alone mesh (simphony.cuds.mesh).  

Is it planned that later on that this will be changed?  For example by (1) replacing `Mesh` by a `FoamMesh` that implements ABCMesh or alternatively (2) by the wrapper keeping track of the created simphonyMesh through multiple simulation runs of open foam.

I guess i am wondering about:
```python
        foam_wrapper = FoamWrapper(self.model)
        meshFoo = foam_wrapper.get_mesh("foo")
        new_point_uid = meshFoo.add_point(Point(coordinates=(0.0, 0.0, 0.0))
 
        #  new point is not really in wrapper..its just in the returned stand-alone mesh
 
        meshFooAgain = foam_wrapper.get_mesh("foo")
        meshFooAgain.get_point(new_point_uid) # fails
```    

I understand that this might not be addressed yet as it is a work-in-progress but was just curious.